simplemonitor/Monitors/gmirror.py

```diff
@@ -0,0 +1,56 @@
+"""
+Gmirror array checks for simplemonitor.
+"""
+
+import subprocess
+
+from .monitor import Monitor, register
+
+
+@register
+class MonitorGmirrorStatus(Monitor):
+    """
+    Check gmirror status for specified device.
+    """
+
+    monitor_type = "gmirror_status"
+
+    def __init__(self, name: str, config_options: dict) -> None:
+        super().__init__(name, config_options)
+        self.array_device = self.get_config_option(
+            "array_device", required_type="str", required=True
+        )
+        self.expected_disks = self.get_config_option(
+            "expected_disks", required_type="int", required=True
+        )
+
+    def run_test(self) -> bool:
+        """
+        Run `gmirror status` for the specified device. Keep the logic simpler
+        by requiring each device and the # of expected disks to be specified
+        separately.
+        """
+
+        run_cmd = ["gmirror", "status", "-gs", self.array_device]
```

Comment:
I followed the example most of the other monitors that make shell calls of not including the full path, but lmk if there's a safer / better way to do this.